loony_dev/agents/coding.py

```
@@ -322,6 +313,26 @@ def _create_pr(self, task: IssueTask, branch: str) -> None:
             )
             raise
 
+    def _generate_pr_body(self, task: IssueTask, branch: str) -> str:
+        """Ask Claude to write a PR body using the issue description and diff."""
+        try:
+            diff = subprocess.check_output(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)